__test__/dynamodb/gameStateDBService.test.ts

```diff
@@ -573,3 +573,32 @@ describe('test resetPlayedTileInteraction', () => {
     expect(await getCurrentPlayedTile(gameId)).toIncludeSameMembers([]);
   });
 });
+
+/* ----------------------------------------------------------------------------
+ * Test startNewGameRound
+ * ------------------------------------------------------------------------- */
+describe('test startNewGameRound', () => {
+  let gameState: GameState;
+  let gameId: string;
+  let prevHand: UserHand[];
+  let prevWall: string[];
+
+  beforeEach(async () => {
+    gameState = await initGameState(FAKE_GAME_ID, CONNECTION_IDS);
+    gameId = gameState.gameId;
+    prevHand = gameState.hands;
+    prevWall = gameState.wall;
+  });
+
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  test('it should change wall & hands and reset currentTurn', async () => {
+    const updatedGameState = await gameStateDBFunctions.startNewGameRound(gameId, CONNECTION_IDS);
+
+    expect(updatedGameState.currentTurn).toStrictEqual(0);
+    expect(updatedGameState.hands).not.toStrictEqual(prevHand);
+    expect(updatedGameState.wall).not.toStrictEqual(prevWall);
+  });
```

Comment:
need to add more tests to demonstrate the wind and dealer are rotated correctly each round

Reply:
For now, WIN_ROUND lambda changes dealer separately by calling changeDealer before. I've added additional assertions to changedealer to test that wind changes when dealer resets

Reply:
Ok, I have changed it so only `startNewGameRound()` changes the dealer/ wind as well. Added assestions in test for it.